ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -123,11 +123,23 @@ message DeleteUserResponse {
 
 // Required authorization: ``HasRight(ParticipantAdmin)``
 message ListUsersRequest {
+
+    // Pagination token to determine the specific page to fetch.
+    string page_token = 2; // TODO:  url-safe base64 encoded protocol buffer,
+
+
+    // Maximum number of results to be returned by the server. Server will return no more than that many results
+    int32 page_size = 3; // NOTE: If the page_size is 0, the server will decide the number of results to be returned.
```

Comment:
Move this note into the comment above to ensure it lands in the docs.

Reply:
Ok